vendor/processors/stripe_objects.py

```diff
@@ -0,0 +1,83 @@
+import stripe
+
+from django.db.models import TextChoices
+# from vendor.models import CustomerProfile, Offer, Invoice, Subscription
+
+stripe.api_key = "sk_test_51LYCNjJHHVfmV6EHPwkh9bogbRyTQFoiGV85yUrQyPFyur3BI2Rjtkhi7XCCNIsPvzOlYTVjzOYmljHPe1X2caIr00uVSfVkmn"
+
```

Comment:
do you want to start moving this to an env var?

Reply:
yeah, remove hard coded key :)